src/benchmarks/micro/corefx/System.Xml.Linq/Perf.XDocument.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Linq;
+using BenchmarkDotNet.Attributes;
+using MicroBenchmarks;
+
+namespace System.Xml.Linq
+{
+    [BenchmarkCategory(Categories.CoreFX)]
+    public class Perf_XDocument
+    {
+        private XDocument _doc;
+
+        [Benchmark]
+        public XDocument Create() => new XDocument();
```

Comment:
not sure if it makes sense to have this. Are you testing anything specific change with this?

Reply:
My change wouldn't affect XDocument, but I assume it would be helpful for measuring any future changes could happen, but agree that XDocument creation logic not  much likely to be changed. Followed test cases from existing XmlDocument test